grammars/supercollider.cson

```diff
@@ -69,7 +69,7 @@
     'match': '^\\s*(\\**[a-z]{1}[a-zA-Z0-9_]+)\\s*\\{'
   }
   {
-    'match': '\\~[a-zA-Z0-9]+'
+    'match': '\\~[a-zA-Z0-9_]+'
```

Comment:
Ah, great catches. This was a quick fix while I was working this morning, but this is more robust. I tested your solution, and it correctly does not highlight `~_`, `~_a`, `~ABC`, or `~900`.

Should I amend it and resubmit with `\\~[a-z][a-zA-Z0-9_]*`?